pointcloud/basic_octree_utils_test.go

```diff
@@ -348,26 +344,32 @@ func TestBasicOctreeCollision(t *testing.T) {
 		// create a non-colliding obstacle near an octree point
 		near, err := spatialmath.NewBox(spatialmath.NewPoseFromPoint(r3.Vector{-2443, 0, 3855}), r3.Vector{1, 2, 3}, "near")
 		test.That(t, err, test.ShouldBeNil)
-		collides, err := basicOct.CollidesWithGeometry(near, 80, 10.0, 1e-8)
+		collides, err := basicOct.CollidesWith(near, 10.0)
 		test.That(t, err, test.ShouldBeNil)
 		test.That(t, collides, test.ShouldBeTrue)
 	})
 
-	t.Run("no collision with box overlapping low-probability octree points", func(t *testing.T) {
-		// create a colliding obstacle overlapping an octree point that has sub-threshold probability
-		lowprob, err := spatialmath.NewBox(spatialmath.NewPoseFromPoint(r3.Vector{-2471, 0, 3790}), r3.Vector{3, 2, 3}, "lowprob")
-		test.That(t, err, test.ShouldBeNil)
-		collides, err := basicOct.CollidesWithGeometry(lowprob, 80, 1.0, 1e-8)
-		test.That(t, err, test.ShouldBeNil)
-		test.That(t, collides, test.ShouldBeFalse)
-	})
-
 	t.Run("collision with box overlapping octree points", func(t *testing.T) {
 		// create a colliding obstacle overlapping an octree point
 		hit, err := spatialmath.NewBox(spatialmath.NewPoseFromPoint(r3.Vector{-2443, 0, 3855}), r3.Vector{12, 2, 30}, "hit")
 		test.That(t, err, test.ShouldBeNil)
-		collides, err := basicOct.CollidesWithGeometry(hit, 80, 1.0, 1e-8)
+		collides, err := basicOct.CollidesWith(hit, 1.)
 		test.That(t, err, test.ShouldBeNil)
 		test.That(t, collides, test.ShouldBeTrue)
 	})
+
+	t.Run("no collision with box overlapping low-probability octree points", func(t *testing.T) {
+		// add a new point to the tree and make sure nothing else can interact with it
+		threshold := 1e-8
+		lowProbPt := r3.Vector{-10000, -10000, -10000}
+		data := NewValueData(0)
```

Comment:
I'm not sure how this was ever working.  The point that we were checking had a confidence of 64 and the threshold is 50.  This makes the test clearer

Reply:
The old test used `collides, err := basicOct.CollidesWithGeometry(lowprob, 80, 1.0, 1e-8)`, manually setting confidence threshold to 80